test/v1/connectors/DeleteUkDividendsIncomeSummaryConnectorSpec.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.connectors
+
+import api.connectors.ConnectorSpec
+import api.mocks.MockHttpClient
+import api.models.domain.{Nino, TaxYear}
+import api.models.outcomes.ResponseWrapper
+import mocks.MockAppConfig
+import play.api.libs.json.Json
+import v1.models.request.deleteUkDividendsIncomeAnnualSummary.DeleteUkDividendsIncomeAnnualSummaryRequest
+
+import scala.concurrent.Future
+
+class DeleteUkDividendsIncomeSummaryConnectorSpec extends ConnectorSpec {
```

Comment:
Maybe rename class and file:
```suggestion
class DeleteUkDividendsIncomeAnnualSummaryConnectorSpec extends ConnectorSpec {
```
so that it matches with the connector that it is testing (and so is easier to find).